DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: RcppMsgPack
 Type: Package
 Title: 'MsgPack' C++ Header Files and Interface Functions for R
-Version: 0.2.2
-Date: 2018-05-06
+Version: 0.2.3
```

Comment:
I would call this 0.2.2.1 for the PR, and only go to 0.2.3 once we're ready for CRAN but no need to change it, I can do this here.